src/popup/router/pages/Dashboard.vue

```diff
@@ -1,98 +1,95 @@
-          :title="$t('dashboard.receive-card.title')"
-          :description="$t('dashboard.receive-card.description')"
-          clickable
-          @click="openTransferReceiveModal()"
-        >
-          <template #icon>
-            <ArrowReceiveIcon />
-          </template>
-        </Card>
-        <Card
-          :title="$t('dashboard.send-card.title')"
-          :description="$t('dashboard.send-card.description')"
-          clickable
-          @click="openTransferSendModal()"
-        >
-          <template #icon>
-            <ArrowSendIcon />
-          </template>
-        </Card>
-      </CardRow>
-      <CardRow v-if="!backedUpSeed">
-        <Card
-          :title="$t('dashboard.back-up-card.title')"
-          :description="$t('dashboard.back-up-card.description')"
-          is-big
-        >
-          <template #icon>
-            <SubtractIcon />
-          </template>
-          <Button
-            class="card-button"
-            :text="$t('dashboard.back-up-card.button')"
-            new-ui
-            inline
-            no-margin
-            :to="{ name: 'settings-seed-phrase' }"
-            fill="red"
-          />
-        </Card>
-      </CardRow>
-      <CardRow>
-        <Card
-          :title="$t('dashboard.buy-card.title')"
-          :description="$t('dashboard.buy-card.description')"
-          is-big
-          :background="buyBackground"
-        >
-          <template #icon>
-            <CardIcon />
-          </template>
-          <Button
-            class="card-button"
-            :text="$t('dashboard.buy-card.button')"
-            new-ui
-            inline
-            no-margin
-            :to="SIMPLEX_URL"
-            fill="red-2"
-          />
-        </Card>
-      </CardRow>
-      <CardRow>
-        <Card
-          :title="$t('dashboard.name-card.title')"
-          :description="$t('dashboard.name-card.description')"
-          is-big
-          :background="chainNameBackground"
-        >
-          <template #icon>
-            <MenuCardIcon />
-          </template>
-          <Button
-            class="card-button"
-            :text="$t('dashboard.name-card.button')"
-            new-ui
-            inline
-            no-margin
-            :to="{ name: 'account-details-names' }"
-            fill="purple"
-          />
-        </Card>
-      </CardRow>
-    </Plate>
+  <div class="dashboard">
+    <AccountSwitcher :notification="!backedUpSeed" />
+    <CardRow class="first-card-wrapper">
+      <Card
+        :title="$t('dashboard.receive-card.title')"
+        :description="$t('dashboard.receive-card.description')"
+        clickable
+        @click="openTransferReceiveModal()"
+      >
+        <template #icon>
+          <ArrowReceiveIcon />
+        </template>
+      </Card>
+      <Card
+        :title="$t('dashboard.send-card.title')"
+        :description="$t('dashboard.send-card.description')"
+        clickable
+        @click="openTransferSendModal()"
```

Comment:
openTransferSendModal

Reply:
Talked about it on priv. I'm leaving it as it is.